snapshot_tests/view/column_view_padding_static_children.scene.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
All the test include children that have specified size and overflow. You have not test that validate if this works for view with undefined size